mne/viz/backends/_pyvista.py

```diff
@@ -151,8 +151,10 @@ def __init__(self, fig=None, size=(600, 600), bgcolor='black',
                          smooth_shading=smooth_shading)
         self.font_family = "arial"
         self.tube_n_sides = 20
-        antialias = _get_3d_option('antialias')
-        self.antialias = antialias and not MNE_3D_BACKEND_TESTING
+        self.antialias = _get_3d_option('antialias') and \
+            not MNE_3D_BACKEND_TESTING
+        self.depth_peeling = _get_3d_option('depth_peeling') and \
+            not MNE_3D_BACKEND_TESTING
```

Comment:
For this stuff, in `conftest.py` we should maybe just have a session-level fixture that picks up on `MNE_3D_BACKEND_TESTING` and `monkeypatch.setenv('MNE_3D_OPTION_ANTIALIAS', 'false')`. Then we don't need code here, just in our tests...

Reply:
The value still needs to be tested in the code right? Howere the variable is set

Reply:
I'm not sure what you mean. 

Currently, `MNE_3D_BACKEND_TESTING=true` is always set if and only if testing is done, right?

If that's the case, then instead of even having this env var, we should `monkeypatch.setattr('MNE_3D_OPTION_ANTIALIAS', 'false')` in a fixture that pytest uses for the entire session. That way the code here can just be:
```
self.antialias = _get_3d_option('antialias')
```
And the `_get_3d_option` will `get_config('MNE_3D_OPTION_ANTIALIAS')`, which will pick up the env var that we set in the fixture via `monkeypatch. The effect ends up being the same: we turn off antialiasing (and depth peeling) in all our tests, and we only mess with the value the user has set during testing.

It helps keep testing-related stuff in `conftest.py` and out of places it doesn't belong (like `_pyvista.py`)

Reply:
... if it's not clear what I mean by this, I can push a quick commit to do what I suggest and you can see if it makes sense

Reply:
Oh now I get it, thanks for the explanation. 

Of course you can commit in here @larsoner

Reply:
@GuillaumeFavelier can you open a follow-up PR for this?